example/decorated/app/src/Decorated.scala

```diff
@@ -40,5 +40,10 @@ object Decorated extends cask.MainRoutes{
     world + user
   }
 
+  @loggedIn()
+  @cask.get("/hello-default")
+  def defaults(world: String = "world")(user: User) = {
```

Comment:
without the fix, this would fail with
```
[error] Decorated.scala:45:7: type mismatch;
[error]  found   : String
[error]  required: app.Decorated.User
[error]   def defaults(world: String = "world")(user: User) = {
[error]       ^
[error] one error found
```